specification/glossary.md

```diff
@@ -46,7 +46,7 @@ A customer's agreement to consume a specific quantity of a service or resource o
 
 <a name="glossary:commitment-based-discount"><b>Commitment-Based Discount</b></a>
 
-Also known as Commitment Discount, this is a commitment for an amount of usage or spend throughout a specified term, in exchange for discounted unit pricing on that amount. The commitment may be based on quantities of resource units or monetary value, with various payment options and time frames.
+A billing discount model that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term.  Commitment-based discount purchases, made upfront and/or with recurring monthly payments are amortized evenly across predefined charge periods (i.e. hourly), and unused amounts cannot be carried over to subsequent charge periods.
```

Comment:
```suggestion
A billing discount model that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term.  Commitment-based discount purchases, made upfront and/or with recurring monthly payments are amortized evenly across predefined charge periods (i.e. hourly), and unused amounts cannot be carried over to subsequent charge periods. Commitment discounts are publicly available to customers without special contract arrangements.
```

By adding this sentence (reference: [FinOps Terminology](https://www.finops.org/assets/terminology/#commitment-discounts)), we can exclude those 'problematic discounts' (e.g., OCI Universal Credits) that I initially classified as Commitment-Based Discounts because they match all other criteria we’ve discussed (e.g. concept of burn-down commitment, purchase amortization, etc.).